interop-cats/jvm/src/main/scala/zio/interop/catszmanagedjvm.scala

```diff
@@ -17,30 +17,31 @@
 package zio
 package interop
 
+import cats.arrow.FunctionK
 import cats.effect.Resource.{ Allocate, Bind, Suspend }
-import cats.effect.{ ExitCase, LiftIO, Resource, Sync, IO => CIO }
+import cats.effect.{ ConcurrentEffect, ExitCase, LiftIO, Resource, Sync, IO => CIO }
 import cats.{ effect, Bifunctor, Monad, MonadError, Monoid, Semigroup, SemigroupK }
 
 trait CatsZManagedSyntax {
   import scala.language.implicitConversions
 
-  implicit final def catsIOResourceSyntax[A](resource: Resource[CIO, A]): CatsIOResourceSyntax[A] =
+  implicit final def catsIOResourceSyntax[F[_]: ConcurrentEffect, A](
```

Comment:
If I am not mistaken, the `ConcurrentEffect` requirement here is redundant, as it is already listed in the `toManaged` method. Could you try removing it?

Reply:
Of course! Fixed